Wikipedia/Code/ReadingListDetailViewController.swift

```diff
@@ -109,21 +104,10 @@ class ReadingListDetailViewController: ColumnarCollectionViewController, Editabl
         navigationBar.addExtendedNavigationBarView(extendedView)
     }
     
-    override func viewWillHaveFirstAppearance(_ animated: Bool) {
-        super.viewWillHaveFirstAppearance(animated)
-        setNavigationBarHidingEnabled(true)
```

Comment:
If you add `setNavigationBarHidingEnabled(true)` to `viewDidLoad`, you'll be able to make sure that the search bar (in this case, `extendedView`) is scrolled away unless it's active (like in `SavedViewController`). Also, feel free to get rid of `setNavigationBarHidingEnabled` and set `navigationBar.isExtendedViewHidingEnabled` directly if you want